Gaming The Syllabus/Assets/Scripts/Combat/UI/AbilityPageUIManager.cs

```diff
@@ -0,0 +1,56 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+using TMPro;
+
+public class AbilityPageUIManager : Singleton<AbilityPageUIManager>
+{
+
+    public GameObject abilityPage;
+    public GameObject abilityButtonPrefab;
+
+    public GameObject abilitiesGameObject;
+
+
+
+    private void Awake()
+    {
+        InitializeSingleton();
+    }
+    // Start is called before the first frame update
+    public IEnumerator Start()
+    {
+        while (!CombatManager.isDone)
+            yield return null;
```

Comment:
Ah yeah I see what's up with isDone now. I'd move the code into another method and have CombatManager call that method when it's ready